.github/workflows/review_cypress_e2e_tests.yml

```diff
@@ -0,0 +1,29 @@
+name: Cypress end-to-end tests
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - backend/**
+      - client/**
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-16.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          browser: chrome
+          headless: true
```

Comment:
if is headless it doesn't have a graphical user interface, what screenshots are going to use?

Reply:
I'm removing this part